src/components/Footer/index.js

```diff
@@ -11,21 +11,76 @@ import {
   SocialMediaLinks,
   FooterLinks,
   Copyright,
+  SocialMediaIcons,
+  FooterButtons,
 } from './StyledFooter'
 import { Col } from '../Grid'
+import WhatsAppImg from '../../assets/Images/whatsapp.svg'
+import PhoneImg from '../../assets/Images/phone.svg'
+import WebChatImg from '../../assets/Images/chat.svg'
```

Comment:
Can we re-use the assets currently on RAD for these three icons (Web Chat, WhatsApp, Contact us)? I'm aware this isn't straightforward because at least two fo those are part of a larger sprite that we wouldn't necessarily want to import to this project. Nonetheless we do need to retain the look and feel of RAD on this, and other directories as far as possible.